### PR TITLE
Added new BrowserStorage module

### PR DESCRIPTION
**Pull request by whitetigle**
Hi there,
since I needed to use sessionStorage in my app I decided to change the BrowserLocalStorage to a more 'generic' BrowserStorage module with two submodules: LocalStorage and SessionStorage.

So in this PR I just added this new module without deleting the old one since many projects still use it.

I don't know if the choice I made is the best one but I'm open to discuss it with you so that we can end up having SessionStorage as well in Fable Powerpack.

Thanks!

### CODE REVIEW

**Comment by MangelMaxime** [comment]
I am in favor of removing the old API. The compiler will detect the error and help users to migrate their code.

Also, this mean we need to release a major version of Fable.Powerpack or at least an "medium" (x.y.z the y number here) version.

File: src/BrowserStorage.fs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module Fable.PowerPack.BrowserStorage
```
Comment (alfonsogarciacaro) [inline src/BrowserStorage.fs]: If we change this line to `namespace Fable.PowerPack` and the module names to `module BrowserLocalStorage` and `module BrowserSessionStorage` we don't need to deprecate the current API.

**Comment by alfonsogarciacaro** [comment]
Adding helpers for SessionStorage is a great idea, thank you! Though I'm not so sure about the API change and the full name (`BrowserStorage.LocalStorage`). Why not just add the `BrowserSessionStorage` module? I added a comment about that.

The new HTTP options look good btw :+1:

**Comment by MangelMaxime** [comment]
Ah yes, @alfonsogarciacaro idea is good to not break the API.

And the HTTP options look good for me too.

**Comment by whitetigle** [comment]
> Why not just add the BrowserSessionStorage module
Well I first started with that. But it seemed weird to have two different namespaces for something somehow  related.

Then I tried `BrowserStorage.Local` and `BrowserStorage.Session` but it did not feel right to me because I always think about `SessionStorage `or `LocalStorage `not `session `or `local`

IMHO, the best would have been `Browser.SessionStorage` and `Browser.LocalStorage` but then we could have another Browser in the namespace. 
`Fable.import.Browser` vs `Fable.Powerpack.Browser`

So I ended up with that naming which, I agree, is not the best one out there 😉

File: src/BrowserStorage.fs (hunk 1 above)
Comment (whitetigle) [reply src/BrowserStorage.fs]: Done 👍

**Pull request merged**